enforcer/src/main/java/org/wso2/micro/gateway/enforcer/security/AuthFilter.java

```diff
@@ -82,6 +83,10 @@ private void initializeAuthenticators(APIConfig apiConfig) {
         }
         Authenticator authenticator = new InternalAPIKeyAuthenticator(APIConstants.JwtTokenConstants.INTERNAL_KEY);
         authenticators.add(authenticator);
+    
+        Authenticator nonSecuredAPIAuthenticator = new NonSecuredAPIAuthenticator();
```

Comment:
Shall we make this class name UnsecuredAPIAuthenticator

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/1784/commits/f109d15dcf8d4806058a4559c61d5cd96408a5c7